app/models/concerns/foreman_remote_execution/host_extensions.rb

```diff
@@ -2,6 +2,8 @@ module ForemanRemoteExecution
   module HostExtensions
     def self.prepended(base)
       base.instance_eval do
+        attr_accessor :template_invocation_status
```

Comment:
Since it is a thing used only in one view in the api, could we pass that to the view in some other way to prevent hosts from always having this attribute?

Reply:
Yeah but in what other way? I was thinking about different solution, but couldn't find anything better than this.

Reply:
If we did what I suggest in the other two comments, we could drop this.